src/compliance/privacy/pi-data-reference-m2.md

```diff
@@ -0,0 +1,324 @@
+
+| Table                      | Column  | Data type     |
+| -------------------------- | ------- | ------------- |
+| `customer_entity_datetime` | `value` | datetime      |
+| `customer_entity_decimal`  | `value` | decimal(12,4) |
+| `customer_entity_int`      | `value` | int(11)       |
+| `customer_entity_text`     | `value` | text          |
+| `customer_entity_varchar`  | `value` | varchar(255)  |
+
+#### `customer_grid_flat` table
+
+The following columns in the `customer_grid_flat` table contain customer information:
+
+| Column               | Data type    |
+| -------------------- | ------------ |
+| `name`               | text         |
+| `email`              | varchar(255) |
+| `dob`                | date         |
+| `gender`             | int(11)      |
+| `shipping_full`      | text         |
+| `billing_full`       | text         |
+| `billing_firstname`  | varchar(255) |
+| `billing_lastname`   | varchar(255) |
+| `billing_telephone`  | varchar(255) |
+| `billing_postcode`   | varchar(255) |
+| `billing_country_id` | varchar(255) |
+| `billing_region`     | varchar(255) |
+| `billing_city`       | varchar(255) |
+| `billing_fax`        | varchar(255) |
+| `billing_vat_id`     | varchar(255) |
+| `billing_company`    | varchar(255) |
+
+### Address data
+
+Magento 2 stores the following customer attributes:
+
+-  City
+-  Company
+-  Country
+-  Fax
+-  First Name
+-  Last Name
+-  Middle Name/Initial
+-  Name Prefix
+-  Name Suffix
+-  Phone Number
+-  State/Province
+-  State/Province ID
+-  Street Address
+-  VAT Number
+-  Zip/Postal Code
+
+#### `customer_address_entity` and reference tables
+
+The following columns in the `customer_address_entity` table contain customer information:
+
+| Column       | Data type    |
+| ------------ | ------------ |
+| `city`       | varchar(255) |
+| `company`    | varchar(255) |
+| `country_id` | varchar(255) |
+| `fax`        | varchar(255) |
+| `firstname`  | varchar(255) |
+| `lastname`   | varchar(255) |
+| `middlename` | varchar(255) |
+| `postcode`   | varchar(255) |
+| `region`     | varchar(255) |
+| `region_id`  | int(10)      |
+| `street`     | text         |
+| `suffix`     | varchar(40)  |
+| `telephone`  | varchar(255) |
+| `vat_id`     | varchar(255) |
+
+These tables reference `customer_address_entity` and can contain custom customer attributes:
+
+| Table                              | Column  | Data type     |
+| ---------------------------------- | ------- | ------------- |
+| `customer_address_entity_datetime` | `value` | datetime      |
+| `customer_address_entity_decimal`  | `value` | decimal(12,4) |
+| `customer_address_entity_int`      | `value` | int(11)       |
+| `customer_address_entity_text`     | `value` | text          |
+| `customer_address_entity_varchar`  | `value` | varchar(255)  |
+
+### Order data
+
+The `sales_order` and related tables contain the customer's name, billing and shipping addresses, and related data.
```

Comment:
```suggestion
The `sales_order` and related tables contain the customer name, billing and shipping addresses, and related data.
```

Reply:
suggest replacing occurrences of "customer's" with "customer"

Reply:
Fixed.